internal/testutil/acc/advanced_cluster.go

```diff
@@ -101,3 +103,29 @@ func CheckExistsClusterHandlingRetry(projectID, clusterName string) error {
 		return nil
 	})
 }
+
+func ConvertToTPFAttrsMap(attrsMap map[string]string) {
```

Comment:
Why not return a new map instead of mutating? Seems cleaner to me

Reply:
that's fair, changed here:  https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2862/commits/4e45b6e500a47a048f6499f115079f6fe746fa6a